TLM/TLM/Patch/_VehicleAI/UpdatePathTargetPositionsPatch.cs

```diff
@@ -69,9 +70,18 @@ public class UpdatePathTargetPositionsPatch {
                            && (GlobalConfig.Instance.Debug.ApiExtVehicleType == API.Traffic.Enums.ExtVehicleType.None
                                || GlobalConfig.Instance.Debug.ApiExtVehicleType == API.Traffic.Enums.ExtVehicleType.RoadVehicle)
                            && (DebugSettings.VehicleId == 0 || DebugSettings.VehicleId == vehicleID);
+            bool debugOverlay = logLogic && DebugSettings.VehicleId != 0;
 #else
-            var logLogic = false;
+            const bool logLogic = false;
+            const bool debugOverlay = false;
 #endif
+            if (debugOverlay) {
+                DebugOverlay.Actions.Remove(0);
```

Comment:
we need to remove these because the overlay needs to clear if vehicle exits node.